articles/reporting/signals-analytics.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Signals Analytics"
+description: "Learn about the detailed reporting and analytics provided about your sending."
+---
+
+One of the major benefits of SparkPost is the vast amount of reporting and analytics provided. With over forty different metrics about the entire lifecycle of your email transmissions, you can understand the success of your program, track key numbers, and find areas to improve your sending program.
+
+## Summary Report
+
+The default reporting view is the Summary report, as seen below. It provides you the ability to select from all metrics SparkPost tracks, adjust reporting date ranges, and filter by campaign, sending domain, recipient domain, and more. [Learn more](./summary) about the Summary report.
```

Comment:
Remove first sentence. 
Alertnative:
"The Summary report lets you pick from any of the metrics SparkPost tracks. You can also filter by campaign, sending domain, recipient domains, and more. [...]"